sklearn/utils/_estimator_html_repr.py

```diff
@@ -376,14 +199,22 @@ def estimator_html_repr(estimator):
     estimator : estimator object
         The estimator to visualize.
 
+    theme : str, optional
```

Comment:
Given there is only one "auto" option to start off with, I prefer not to introduce a public API.